Tasks/Src/RTOS_BPS.c

```diff
@@ -11,6 +11,20 @@ void RTOS_BPS_SemPost(void) {
 
 }
 
+/**
+ * @brief   Waits for Mutex, assigns timestamp and any error to err and ticks
+ * @param   *mutex - pointer to mutex
+ * @param   timeout - timeout period, if 0 will wait forever until resource available
+ * @param   options - determines if mutex available or not
```

Comment:
Is this comment correct?

Reply:
yes i think so

Reply:
no it aint, look at the options parameter in the micrium function.